storage/tests/unit/test_notification.py

```diff
@@ -16,6 +16,72 @@
 
 import mock
 
+class TestProjectID(unittest.TestCase):
+
+    from google.cloud.storage import notification
```

Comment:
1. Please have **two** empty lines between top-level definitions (i.e. after `import mock` and before `class TestProjectID`)
1. This import will still happen at the time this module is imported, not at runtime, so please move it.